src/mv.js

```diff
@@ -110,6 +110,8 @@ function _mv(options, sources, dest) {
         // exception to pass up to the top level.
         cp('-r', src, thisDest);
         rm('-rf', src);
+      } else {
+        common.error(e.message, { continue: true });
```

Comment:
Could you catch a specific `e.code` and output a message like `mv: cannot move 'foo/a/file.js' to 'bar/a/file.js': No such file or directory`?

Also, could you add an else branch with `throw e`? The distinction is that we only want to use `common.error()` for expected error conditions, and would prefer to fail loudly with an exception for unexpected conditions (so that we can fix these as they're reported).